docs/users.rst

```diff
@@ -490,6 +490,16 @@ version
 This key is no longer used by OSBS and is only kept in the schema for backwards
 compatibility.
 
+
+set_release_env
+~~~~~~~~~~~~~~~
+
+Optional string.  If set, the image's Dockerfile will be amended with
+"ENV $set_release_env=$RELEASE" and an environment variable with of the string
```

Comment:
"ENV $set_release_env=$RELEASE": it might read as though we add that literal string, which of course we don't. Can we say "an ENV statement"? The words afterwards clarify what the statement does.

"with of": some words missing?

Reply:
Rewritten, thanks for the review.